### DISCUSSION TITLE
can uwebsockets be compared to nchan and how ?

### DISCUSSION THREAD

**Discussion by kapouer** [uNetworking/uWebSockets#1205] [General]
Hi,
i suppose uWebSockets is going to be better than nchan,
but how exactly ? It would be nice to know !
https://github.com/slact/nchan

**Comment by ghost** [comment]
> powered by Nginx.

We have NGINX benchmarks in benchmark folder. We are roughly 3x that of NGINX, but NGINX is also a lot more used and well-known and feature packed so I can't really say what is "best" or not.
